configserver/src/main/resources/configserver-app/services.xml

```diff
@@ -102,8 +102,7 @@
       <binding>http://*/status</binding>
     </handler>
     <handler id='com.yahoo.vespa.config.server.http.v2.TenantHandler' bundle='configserver'>
-      <binding>http://*/application/v2/tenant</binding>
-      <binding>http://*/application/v2/tenant/*</binding>
+      <binding>http://*/application/v2/tenant*</binding>
```

Comment:
I'm surprised this pattern has not been commonly used before (instead of having the two previous bindings)

Reply:
Yes. It doesn't work well for extracting the matched groups, as those now contain the `/` in sub paths, but these bindings aren't used for that. I suspect that's the reason, though.